### PR TITLE
Add clear and grant permsissions tasks

### PR DESCRIPTION
**Pull request by ankur22**
## What?

Update the next release notes with links to the PRs that have refactored `clearPermissions` and `grantPermissions`, as well as the fix for `clearPermissions`.

## Why?

To update the release notes for the next release.

## Checklist

- [x] I have performed a self-review of my code.
- [ ] I have added tests for my changes.
- [x] I have run linter locally (`make lint`) and all checks pass.
- [x] I have run tests locally (`make tests`) and all tests pass.
- [ ] I have commented on my code, particularly in hard-to-understand areas.

### CODE REVIEW

File: release notes/v0.48.0.md [hunk 1]
```diff
@@ -33,13 +33,11 @@ _Format as `<number> <present_verb> <object>. <credit>`_:
 
 ## Bug fixes
 
-_Format as `<number> <present_verb> <object>. <credit>`_:
-- _`#111` fixes race condition in runtime_
+- [browser#1077](https://github.com/grafana/xk6-browser/pull/1077) Fixes `browserContext.clearPermissions` so it no longer panics and actually performs the clear permissions action in the chrome browser.
```
Comment (inancgumus) [inline release notes/v0.48.0.md:36]: This is OK, but what about the following? Should we also link to the documentation like we did in previous release notes?

```suggestion
- [browser#1077](https://github.com/grafana/xk6-browser/pull/1077) Fixes `browserContext.clearPermissions` to clear permissions without panic.
```

**Review by inancgumus — APPROVED** [review]

**Review by ka3de — APPROVED** [review]
LGTM.

**Review by mstoykov — APPROVED** [review]

File: release notes/v0.48.0.md (hunk 1 above)
Comment (ankur22) [reply release notes/v0.48.0.md:36]: I'm not sure we need to link to documentation for fixes like this, I can see value in it if there's a new feature though or a breaking change but i guess it depends on a case-by-case basis. I like the suggestion you've made though 👍

Comment (inancgumus) [reply release notes/v0.48.0.md]: Sure, no worries.

**Pull request merged**